lib/iris/tests/test_analysis.py

```diff
@@ -365,6 +365,26 @@ def test_percentile_1d(self):
                        ('analysis', 'third_quartile_foo_1d.cml'),
                        checksum=False)
 
+    def test_fast_percentile_1d(self):
+        cube = tests.stock.simple_1d()
+
+        first_quartile = cube.collapsed('foo', iris.analysis.PERCENTILE,
+                                        percent=25, percentile_method='fast')
+        np.testing.assert_array_almost_equal(first_quartile.data,
+                                             np.array([2.5], dtype=np.float32))
+        self.assertCML(first_quartile, ('analysis',
```

Comment:
Also, I just noticed that nothing here is testing for multiple values passed to the percentile arg.

Actually TBH these are very old tests, as are any not under `iris/tests/unit` or `iris/tests/integration`.
I don't have time jst now to research a more definite suggestion, but I would definitely be thinking of adding new tests into `iris.tests.unit.analysis.test_PERCENTILE` or `iris.tests.unit.analysis.test_PercentileAggregator`.
There is testing there for multiple values, too.

Sorry to have been so slow seeing the obvious here !